player-ui/src/main/res/drawable/ic_pause_circle.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:tint="#FFFFFF"
+    android:viewportWidth="24.0"
+    android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```

Comment:
For things like colors and strings, I think it's worth defining in a shared module. 

Shall I flip the dependencies such that `app` is a base module? (separate PR)

Reply:
yes pleaseee sounds good